examples/semantic_segmentation/README.md

```diff
@@ -0,0 +1,38 @@
+# Examples of Semantic Segmentation
+
+
+## Supported models
+- SegNet
+- PSPNet
+
+For the details, please check the documents and examples of each model.
+
+## Performance
+
+The scores are mIoU.
+
+### Cityscapes
+
+| Model | Reference | ChainerCV (weight conversion) |
+|:-:|:-:|:-:|
+| PSPNet with ResNet101 (single scale) | 79.70 % [1] | 79.03 % |
+
+```
+$ python eval_cityscapes.py [--model pspnet_resnet101] [--gpu <gpu>] [--pretrained-model <model_path>]
+```
+
+### CamVid
+
+| Model | Reference | ChainerCV |
+|:-:|:-:|:-:|
+| SegNet | 46.3 % [2] | 49.4 % |
+
+```
+$ python eval_camvid.py [--gpu <gpu>] [--pretrained_model <model_path>] [--batchsize <batchsize>]
```

Comment:
`--pretrained_model` -> `--pretrained-model`

Reply:
The order of arguments should be consistent with `eval_cityscapes.py` (Modifying `eval_cityscapes.py` is also ok)

Reply:
You mean making them consistent with `eval_detection.py`?

I don't fine the problem you are mentioning. The order of `gpu` and `pretrained-model` are the same between the two under `examples/semantic_segmentation`.

Reply:
Sorry. I misunderstood. You were talking about the inconsistency  between the script and the README.